docs/CoPMeetingMins/SupportingDocs/agency_liaisons.md

```diff
@@ -25,7 +25,7 @@
 |Nuclear Regulatory Commission |Arthur Smith |arthur.smith@nrc.gov |
 |Office of Personnel Management |Adrian Lintz |adrian.lintz@opm.gov |
 |Small Business Administration |Andrew Davy, Ryan Hillard |Andrew.Davy@sba.gov, Ryan.Hillard@sba.gov |
-|Social Security Administration |Joseph Amalfitano |joseph.amalfitano@ssa.gov |
+|Social Security Administration |Joseph Amalfitano |joseph.amalfitano@ssa.gov, kailash.pathak@ssa.gov |
```

Comment:
@JosephAmalfitanoSSA, do you want to add his full name too? Look at the SBA example above yours.

Reply:
ah - didnt notice. thanks